h/buildext.py

```diff
@@ -219,80 +236,78 @@ def build_chrome(args):
     subprocess.call(subprocess_args)
 
     # Render the sidebar html
-    base_url = env['request'].route_url('index')
-    api_url = env['request'].route_url('api')
-    websocket_url = client.websocketize(env['request'].route_url('ws'))
-    sentry_dsn = env['request'].sentry.get_public_dsn('https')
-    settings = settings_dict(base_url=base_url,
-                             api_url=api_url,
-                             sentry_dsn=sentry_dsn)
+    api_url = '{}/api'.format(base_url)
```

Comment:
This previously had a trailing slash on it. I've had a quick look and I don't think this change should make any difference to how the client works, but wanted to call that out in case you knew better than me.